<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamerun</class>
 <widget class="QDialog" name="gamerun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="stocksTable">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>731</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buySellBtn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>500</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Shares (Buy/Sell)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buySubmitBtn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>500</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sellSubmitBtn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>550</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sell</string>
   </property>
  </widget>
  <widget class="QLabel" name="stockNameLbl">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>560</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="buySellLbl">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>560</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stockDropdown">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>520</y>
     <width>79</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>490</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Stock</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="cashBalanceLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="profitsLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Skip To Next Day</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
